vaadin-testbench-shared/src/main/java/com/vaadin/testbench/annotations/RunLocally.java

```diff
@@ -29,8 +28,6 @@
  * {@code com.vaadin.testbench.Parameters.runLocally} system parameter.
  * <p>
  * This parameter will be used to instantiate the {@link WebDriver}.<br>
- * The value can be obtained through {@link ParallelTest#getRunLocallyBrowser()}
```

Comment:
Would it be helpful to reference `RunLocally` from `ParallelTest`?

Reply:
It is already there in few places:
```
/**
     * <p>
     * Sets the driver for this test instance. Uses
     * {@link SetupDriver#setupRemoteDriver(String)} or
     * {@link SetupDriver#setupLocalDriver(Browser)} according to the
     * annotations found in current test case.
     * </p>
     * <p>
     * {@link RunOnHub} annotation can be used on the test case class to define
     * a test hub's hostname for the driver to connect to it.<br>
     * {@link RunLocally} annotation can be used on the test case class to force
     * the driver to connect to localhost ({@link RunLocally} annotation
     * overrides {@link RunOnHub} annotation).
     * </p>
     *
     * @throws Exception
     *             if unable to instantiate {@link WebDriver}
     */
```
or
```
/**
     * @return Browser value of the {@link RunLocally} annotation of current
     *         Class, or null if annotation is not present.
     */
    protected Browser getRunLocallyBrowser() {
        return ParallelRunner.getRunLocallyBrowserName(getClass());
    }
```